### PR TITLE
no message

### PR DESCRIPTION
**Pull request by chowdermcarthor**
more things

### CODE REVIEW

File: code/game/jobs/job/bunker.dm [hunk 1]
```diff
@@ -168,6 +171,7 @@
 	no_id = 1
 	no_pda= 1
 	no_headset = 1
+	duties = "You fight and protect whoever pays you the most. Fighting is what you do, and you excell at it."
```
Comment (mattroks101) [inline code/game/jobs/job/bunker.dm]: One too may "L"s on "excel".

File: vgstation13.dme [hunk 2]
```diff
@@ -1174,17 +1180,6 @@
 #include "code\modules\hydroponics\spreading\spreading.dm"
 #include "code\modules\hydroponics\spreading\spreading_growth.dm"
 #include "code\modules\hydroponics\spreading\spreading_response.dm"
-#include "code\modules\interactions\_interaction.dm"
```
Comment (mattroks101) [inline vgstation13.dme]: This shouldn't be un-included.

File: code/game/machinery/door_control_announce.dm [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+				return
+		var/datum/speech/speech = announcement_intercom.create_speech("The Bunker Overseer has decided to [function_open ? "[activated ? "[function_open]" : "[function_close]"]" : "[activated ? "open" : "close"]"] [opening].", transmitter=announcement_intercom)
+		speech.name = "Dylatov Bunker Announcement Server"
+		speech.job = "Automated Announcement"
+		speech.as_name = "Dylatov Bunker Announcement Server"
+		speech.frequency = freq
+
+		Broadcast_Message(speech, vmask=null, data=0, compression=0, level=list(0,1))
+		returnToPool(speech)
+	return
```
Comment (DrCelt) [inline code/game/machinery/door_control_announce.dm]: Trailing return

File: code/modules/library/lib_items.dm [hunk 4]
```diff
@@ -13,21 +13,28 @@
...
+	var/list/icons_avalible = list()
 	var/list/valid_types = list(/obj/item/weapon/book, \
 								/obj/item/weapon/tome, \
 								/obj/item/weapon/spellbook, \
 								/obj/item/weapon/storage/bible)
 
+/obj/structure/bookcase/New()
+	..()
+	update_icon()
+	return
```
Comment (DrCelt) [inline code/modules/library/lib_items.dm]: Trailing return

File: code/modules/paperwork/photocopier.dm [hunk 5]
```diff
@@ -38,6 +41,15 @@
...
 	RefreshParts()
 
+/obj/machinery/photocopier/update_icon()
+	..()
+	overlays.Cut()
+	if(copy)
+		overlays += "printer_large_scan"
+	if(paper)
+		overlays += "printer_large_paper"
+	return
```
Comment (DrCelt) [inline code/modules/paperwork/photocopier.dm]: Trailing return

File: code/modules/organs/organ_external.dm [hunk 6]
```diff
@@ -1153,10 +1162,17 @@ Note that amputating the affected organ does in fact remove the infection from t
 	var/g = "m"
```
Comment (DrCelt) [inline code/modules/organs/organ_external.dm]: Why is this a variable when owner.gender is a variable?

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 7]
```diff
@@ -879,12 +904,21 @@ var/global/list/damage_icon_parts = list()
...
+				to_chat(src, "<span class='warning'>The [wear_suit] is much too big for you, and sags to the ground.</span>")
+				drop_from_inventory(wear_suit)
+				return
+		else if((((M_FAT in mutations) && (species.flags & CAN_BE_FAT)) || (species.flags & IS_BULKY)) && !(wear_suit.icon_override))
+			if(wear_suit.flags&ONESIZEFITSALL || wear_suit.kidspecial != 2)
 				standing.icon	= 'icons/mob/suit_fat.dmi'
 			else
 				to_chat(src, "<span class='warning'>You burst out of \the [wear_suit]!</span>")
 				drop_from_inventory(wear_suit)
+		else if(gender == "female" && w_uniform.fitted && species.female_icons)
```
Comment (DrCelt) [inline code/modules/mob/living/carbon/human/update_icons.dm]: Female should be a define

File: code/modules/organs/organ_external.dm [hunk 8]
```diff
@@ -1153,10 +1162,17 @@ Note that amputating the affected organ does in fact remove the infection from t
...
 	if(owner.gender == FEMALE)
 		g = "f"
-
+	var/child = 0
+	if(owner.age < AGE_TEEN && owner.species.children)
```
Comment (DrCelt) [inline code/modules/organs/organ_external.dm]: Copy pasting is bad. Use a proc "ischild" for this instead

File: code/modules/paperwork/photocopier.dm [hunk 9]
```diff
@@ -260,28 +285,43 @@
...
 /obj/machinery/photocopier/attackby(obj/item/O, mob/user)
 	if(copying)
 		if(!alert("Cancel current print job?","","Yes","No") == "Yes")
 			to_chat(usr, "<span class='warning'>Must wait for current print job to finish.</span>")
 			return
 		copying = 0
+	if(istype(O, /obj/item/weapon/paper_pack))
+		var/obj/item/weapon/paper_pack/P = O
+		paper += P.amount
+		visible_message("<span>[user] inserts [P.amount] sheets of paper into [src]'s paper storage.</span>")
```
Comment (DrCelt) [inline code/modules/paperwork/photocopier.dm]: Shouldn't this be "\the [src] "?

File: code/modules/paperwork/photocopier.dm [hunk 10]
```diff
@@ -260,28 +285,43 @@
...
-				copy = O
-				to_chat(user, "<span class='notice'>You insert [O] into [src].</span>")
-				flick("bigscanner1", src)
-				updateUsrDialog()
+		var/obj/item/weapon/paper/P = O
+		if(P.info)
+			if(copier_empty())
+				if(user.drop_item(O, src))
+					copy = O
+					to_chat(user, "<span class='notice'>You insert [O] into [src].</span>")
```
Comment (DrCelt) [inline code/modules/paperwork/photocopier.dm]: Shouldn't this be "\the [src] "?

File: code/modules/paperwork/photocopier.dm [hunk 11]
```diff
@@ -260,28 +285,43 @@
...
+					updateUsrDialog()
+			else
+				to_chat(user, "<span class='notice'>There is already something in [src].</span>")
 		else
-			to_chat(user, "<span class='notice'>There is already something in [src].</span>")
+			if(paper >= max_paper)
+				to_chat(user, "<span class='notice'>There's too much spare paper in [src].</span>")
+			else
+				paper ++
+				visible_message("<span>[user] inserts a sheet of paper into [src]'s paper storage.</span>")
```
Comment (DrCelt) [inline code/modules/paperwork/photocopier.dm]: Shouldn't this be "\the [src] "?

File: code/modules/paperwork/photocopier.dm [hunk 12]
```diff
@@ -260,28 +285,43 @@
...
+					copy = O
+					to_chat(user, "<span class='notice'>You insert [O] into [src].</span>")
+//					flick("bigscanner1", src)
+					updateUsrDialog()
+			else
+				to_chat(user, "<span class='notice'>There is already something in [src].</span>")
 		else
-			to_chat(user, "<span class='notice'>There is already something in [src].</span>")
+			if(paper >= max_paper)
+				to_chat(user, "<span class='notice'>There's too much spare paper in [src].</span>")
```
Comment (DrCelt) [inline code/modules/paperwork/photocopier.dm]: Shouldn't this be "\the [src] "?

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 13]
```diff
@@ -879,12 +904,21 @@ var/global/list/damage_icon_parts = list()
...
-			if(wear_suit.flags&ONESIZEFITSALL)
+		if(age < AGE_TEEN && species.children) //No fat kids
+			if(wear_suit.kidspecial)
+				standing.icon	= 'icons/mob/children/suit.dmi'
+			else
+				to_chat(src, "<span class='warning'>The [wear_suit] is much too big for you, and sags to the ground.</span>")
+				drop_from_inventory(wear_suit)
+				return
+		else if((((M_FAT in mutations) && (species.flags & CAN_BE_FAT)) || (species.flags & IS_BULKY)) && !(wear_suit.icon_override))
+			if(wear_suit.flags&ONESIZEFITSALL || wear_suit.kidspecial != 2)
```
Comment (DrCelt) [inline code/modules/mob/living/carbon/human/update_icons.dm]: 2? What is 2? Make this a define.

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 14]
```diff
@@ -608,7 +624,10 @@ var/global/list/damage_icon_parts = list()
 		var/image/standing	= image("icon" = ((gloves.icon_override) ? gloves.icon_override : 'icons/mob/hands.dmi'), "icon_state" = "[t_state]")
 
 		var/obj/item/I = gloves
-
+		if(age < AGE_TEEN && species.children && I.kidspecial)
+			standing.icon = 'icons/mob/children/hands.dmi'
+		else if(gender == "female" && w_uniform.fitted && species.female_icons)
```
Comment (DrCelt) [inline code/modules/mob/living/carbon/human/update_icons.dm]: Female should be a define

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 15]
```diff
@@ -846,7 +868,10 @@ var/global/list/damage_icon_parts = list()
 		var/image/standing = image("icon" = ((belt.icon_override) ? belt.icon_override : 'icons/mob/belt.dmi'), "icon_state" = "[t_state]")
 
 		var/obj/item/I = belt
-
+		if(age < AGE_TEEN && species.children && I.kidspecial)
+			standing.icon = 'icons/mob/children/hands.dmi'
+		else if(gender == "female" && w_uniform.fitted && species.female_icons)
```
Comment (DrCelt) [inline code/modules/mob/living/carbon/human/update_icons.dm]: Female should be a define

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 16]
```diff
@@ -846,7 +868,10 @@ var/global/list/damage_icon_parts = list()
 		var/image/standing = image("icon" = ((belt.icon_override) ? belt.icon_override : 'icons/mob/belt.dmi'), "icon_state" = "[t_state]")
 
 		var/obj/item/I = belt
-
+		if(age < AGE_TEEN && species.children && I.kidspecial)
```
Comment (DrCelt) [inline code/modules/mob/living/carbon/human/update_icons.dm]: I'm noticing this and the female if being copypasted. Make these two into procs, maybe?

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 17]
```diff
@@ -516,14 +524,22 @@ var/global/list/damage_icon_parts = list()
...
+				drop_from_inventory(w_uniform)
+				return
+		else if(((M_FAT in mutations) && (species.flags & CAN_BE_FAT)) || species.flags & IS_BULKY)
+			if(w_uniform.flags&ONESIZEFITSALL || w_uniform.kidspecial != 2)
 				standing.icon	= 'icons/mob/uniform_fat.dmi'
 			else
 				to_chat(src, "<span class='warning'>You burst out of \the [w_uniform]!</span>")
 				drop_from_inventory(w_uniform)
 				return
+		else if(gender == "female" && w_uniform.fitted && species.female_icons)
```
Comment (DrCelt) [inline code/modules/mob/living/carbon/human/update_icons.dm]: Copy pasting is bad!

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 18]
```diff
@@ -516,14 +524,22 @@ var/global/list/damage_icon_parts = list()
...
-			if(w_uniform.flags&ONESIZEFITSALL)
+		if(age < AGE_TEEN && species.children) //No fat kids
+			if(w_uniform.kidspecial)
+				standing.icon	= 'icons/mob/children/uniform.dmi'
+			else
+				to_chat(src, "<span class='warning'>The [w_uniform] is much too big for you, and sags to the ground.</span>")
+				drop_from_inventory(w_uniform)
+				return
+		else if(((M_FAT in mutations) && (species.flags & CAN_BE_FAT)) || species.flags & IS_BULKY)
+			if(w_uniform.flags&ONESIZEFITSALL || w_uniform.kidspecial != 2)
```
Comment (DrCelt) [inline code/modules/mob/living/carbon/human/update_icons.dm]: 2 needs to be a define

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 19]
```diff
@@ -281,6 +284,9 @@ var/global/list/damage_icon_parts = list()
 		if(!fat && !skeleton)
 			stand_icon.Blend(new /icon('icons/mob/human.dmi', "underwear[underwear]_[g]_s"), ICON_OVERLAY)
 
+	if(age < AGE_TEEN && species.children)
```
Comment (DrCelt) [inline code/modules/mob/living/carbon/human/update_icons.dm]: I mean honestly why not just have an ischild proc?

File: code/modules/clothing/under/bunker.dm [hunk 20]
```diff
@@ -5,3 +5,12 @@
...
 	flags = FPRINT  | ONESIZEFITSALL
+
+/obj/item/clothing/under/orphan
+	name = "orphan clothing"
+	desc = "Clothing worn by an orphan."
+	icon_state = "orphan"
+	item_state = "orphan"
+	_color = "orphan"
+	flags = FPRINT  | ONESIZEFITSALL
+	kidspecial = 2
```
Comment (DrCelt) [inline code/modules/clothing/under/bunker.dm]: AaAaaaaaaAAAA DEFINE THIS

File: code/game/machinery/door_control_announce.dm [hunk 21]
```diff
@@ -0,0 +1,60 @@
...
+	opening = "the Armory"
+	id_tag = "armory"
+	required_alert_level = "red"
+	freq = 1359
+
+/obj/machinery/door_control/announced/attack_hand(mob/user as mob)
+	..()
+	activated = !activated
+	Announce(user)
+	return
```
Comment (DrCelt) [inline code/game/machinery/door_control_announce.dm]: Trailing return

File: code/modules/library/lib_items.dm [hunk 22]
```diff
@@ -33,7 +33,7 @@
 /obj/structure/bookcase/New()
 	..()
 	update_icon()
-	return
+//	return
```
Comment (Zuhayr) [inline code/modules/library/lib_items.dm]: Why...?

File: code/game/jobs/job_controller.dm [hunk 23]
```diff
@@ -351,6 +351,14 @@ var/global/datum/controller/occupations/job_master
 
 	H.job = rank
 
+	if(H.age > job.min_age)
```
Comment (DrCelt) [inline code/game/jobs/job_controller.dm]: Should use a clamp here.

File: code/game/objects/items/weapons/storage/belt.dm [hunk 24]
```diff
@@ -7,7 +7,7 @@
 	flags = FPRINT
 	slot_flags = SLOT_BELT
 	attack_verb = list("whips", "lashes", "disciplines")
-
+	kidspecial = 1
```
Comment (DrCelt) [inline code/game/objects/items/weapons/storage/belt.dm]: Defines.

File: code/modules/clothing/clothing.dm [hunk 25]
```diff
@@ -376,13 +381,15 @@ BLIND     // can't see anything
 	slot_flags = SLOT_NECK
 	species_restricted = list("exclude")
 	heat_conductivity = NECK_HEAT_CONDUCTIVITY
+	kidspecial = 1
```
Comment (DrCelt) [inline code/modules/clothing/clothing.dm]: Define.

File: code/modules/organs/organ_external.dm [hunk 26]
```diff
@@ -854,12 +854,21 @@ Note that amputating the affected organ does in fact remove the infection from t
 	if(gender)
 		gender="_[gender]"
 	var/fat = ""
-	if(isFat && has_fat)
-		fat = "_fat"
+	var/child = 0
```
Comment (DrCelt) [inline code/modules/organs/organ_external.dm]: var/child = ischild()

**Comment by mattroks101** [comment]
I'll fix what's left myself.

**Pull request merged**